src/bundle/Resources/views/layout.html.twig

```diff
@@ -34,6 +34,13 @@
         </script>
         <script src="/bundles/fosjsrouting/js/router.js"></script>
         <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
+        {% if app.request.locale == 'ach_UG' %}
```

Comment:
This could be abstracted so that the way to detect in-context isn't bound to the implementation (`if app.in_context_translation_enabled`).